website/www/site/layouts/shortcodes/table_with_icons.html

```diff
@@ -0,0 +1,51 @@
+{{/*
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License. See accompanying LICENSE file.
+*/}}
+{{ $data := index $.Site.Data .Site.Language.Lang }}
+
+<div class="table-wrapper">
+    <table>
+        <tr>
+            <th>PURPOSE</th>
+            <th>CONTACT</th>
+            <th>ACTION</th>
+        </tr>
+        {{ range $item := $data.contact_us }}
+            <tr>
+                <td>
+                    <div>
+                        {{ with resources.Get .icon }}
+                            {{ .Content | safeHTML }}
+                        {{ end }}
+                        <p>{{.purpose | safeHTML}}</p>
```

Comment:
Nit: Could you add spaces before and after curly braces for consistency? Like `<p>{{ .purpose | safeHTML }}</p>`. Maybe there's some hugo linter available that would help with that?

Reply:
not really... I founded only some snippetts